core/new-gui/src/app/workspace/component/workflow-editor/workflow-editor.component.ts

```diff
@@ -90,6 +91,55 @@ export class WorkflowEditorComponent implements AfterViewInit {
     );
   }
 
+  private handleCellHighlight(): void {
+    this.handleHighlightMouseInput();
+    this.handleOperatorHightlightEvent();
```

Comment:
why not move this function to joint-graph-wrapper.ts

Reply:
This is the correct place to put these logic.

The Component is about the "View", where we control the things that manipulate the HTML and CSS. This Workflow Editor Component includes the JointJS `Paper` object, which is corresponds to the view.

The Service is about the "Model", where we keep variables about the state. In JointGraphWrapper, we keep the JointJS `Graph` object. In this case, we manage the state of which operator is highlighted.

Events coming from multiple places can affect the model, in our current code:
  - From the Component View: Clicking on the cell itself will trigger highlighting the operator in the model
  - From Drag and Drop: Dropping the operator will trigger the highlight

The changes in the model will then affect the view - whenever the model changes, the view of which operator is highlighted / unhighlighted must also be changed by manipulating the CSS (in this case, we manipulate CSS through JointJS API).

In summary:
`handleHighlightMouseInput` function manages triggering the change in the model when the user clicks an operator.

`handleOperatorHightlightEvent` function manages the view reacting to changes in the model - whenever an operator is highlighted / unhighlighted, the view must be changed accordingly.

This design about separating model and view is essential to the codebase.